WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3206,6 +3274,17 @@ private void refreshEditorTheme() {
         mDispatcher.dispatch(EditorThemeActionBuilder.newFetchEditorThemeAction(payload));
     }
 
+    private void fetchMediaList() {
+        // do not refresh if there is no network
+        if (!NetworkUtils.isNetworkAvailable(this)) {
```

Comment:
Just a note here, I realized that I don't see many usages of this approach where we check to see if network connectivity exists before utilizing the `dispatcher`. Do you know why this is so?

Reply:
I think I thought of doing something else in this case but, since we don't really need to do anything in particular in this regard (it's just a background refresh), we could also just attempt and let it fail. No big deal 🤷

Reply:
Okay true! 😄